O/OpenFOAM_com/build_tarballs.jl

```diff
@@ -0,0 +1,577 @@
+    ExecutableProduct("surfaceSplitByTopology", :surfaceSplitByTopology),
+    ExecutableProduct("surfaceSplitNonManifolds", :surfaceSplitNonManifolds),
+    ExecutableProduct("surfaceSubset", :surfaceSubset),
+    ExecutableProduct("surfaceToFMS", :surfaceToFMS),
+    ExecutableProduct("surfaceToPatch", :surfaceToPatch),
+    ExecutableProduct("surfaceTransformPoints", :surfaceTransformPoints),
+    ExecutableProduct("surfactantFoam", :surfactantFoam),
+    ExecutableProduct("temporalInterpolate", :temporalInterpolate),
+    ExecutableProduct("tetgenToFoam", :tetgenToFoam),
+    ExecutableProduct("tetMesh", :tetMesh),
+    ExecutableProduct("thermoFoam", :thermoFoam),
+    ExecutableProduct("topoSet", :topoSet),
+    ExecutableProduct("transformPoints", :transformPoints),
+    ExecutableProduct("twoLiquidMixingFoam", :twoLiquidMixingFoam),
+    ExecutableProduct("twoPhaseEulerFoam", :twoPhaseEulerFoam),
+    ExecutableProduct("uncoupledKinematicParcelDyMFoam", :uncoupledKinematicParcelDyMFoam),
+    ExecutableProduct("uncoupledKinematicParcelFoam", :uncoupledKinematicParcelFoam),
+    ExecutableProduct("viewFactorsGen", :viewFactorsGen),
+    ExecutableProduct("vtkUnstructuredToFoam", :vtkUnstructuredToFoam),
+    ExecutableProduct("wallFunctionTable", :wallFunctionTable),
+    ExecutableProduct("writeMeshObj", :writeMeshObj),
+    ExecutableProduct("writeMorpherCPs", :writeMorpherCPs),
+    ExecutableProduct("XiDyMFoam", :XiDyMFoam),
+    ExecutableProduct("XiEngineFoam", :XiEngineFoam),
+    ExecutableProduct("XiFoam", :XiFoam),
+    ExecutableProduct("zipUpMesh", :zipUpMesh),
+    
+    FileProduct("share/openfoam/etc", :openfoam_etc), 
+    FileProduct("share/openfoam/bin", :openfoam_bin)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="flex_jll", uuid="48a596b8-cc7a-5e48-b182-65f75e8595d0"))
+    Dependency(PackageSpec(name="SCOTCH_jll", uuid="a8d0f55d-b80e-548d-aff6-1a04c175f0f9"); compat=SCOTCH_VERSION)
+    Dependency(PackageSpec(name="FFTW_jll", uuid="f5851436-0d7a-5f13-b9de-f02708fd171a"); compat=FFTW_VERSION)
+    Dependency(PackageSpec(name="PTSCOTCH_jll", uuid="b3ec0f5a-9838-5c9b-9e77-5f2c6a4b089f"))
+    Dependency(PackageSpec(name="METIS_jll", uuid="d00139f3-1899-568f-a2f0-47f597d42d70"))
+    Dependency(PackageSpec(name="Zlib_jll", uuid="83775a58-1f1d-513f-b197-d71354ab007a"))
+    Dependency(PackageSpec(name="CGAL_jll", uuid="8fcd9439-76b0-55f4-a525-bad0597c05d8"))
+    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
+    RuntimeDependency(PackageSpec(name = "M4_jll", uuid = "9051c120-a745-5e86-aaa7-8cbc404dba28"))
+]
+append!(dependencies, platform_dependencies)
+
+# Build the tarballs, and possibly a `build.jl` as well.
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; 
+    preferred_gcc_version = v"12")
```

Comment:
https://github.com/JuliaPackaging/Yggdrasil/blob/f06c3a47a4773cdeaa3bd74f7930746e4a8cc553/CONTRIBUTING.md#compatibility-tips

Reply:
Okay. I shall try with v"8", because the minimum required `gcc` version is 7.5.0. In that case the 7.1.0 version in BinaryBuilder will not work.